src/System.Net.Http/tests/FunctionalTests/HttpClientTestBase.cs

```diff
@@ -31,6 +31,11 @@ static HttpClientTestBase()
 
         protected virtual bool UseManagedHandler => false;
 
+        protected bool IsWinHttpHandler => !UseManagedHandler && PlatformDetection.IsWindows && !PlatformDetection.IsUap && !PlatformDetection.IsFullFramework;
+        protected bool IsCurlHandler => !UseManagedHandler && !PlatformDetection.IsWindows;
+        protected bool IsNetfxHandler => !UseManagedHandler && PlatformDetection.IsWindows && PlatformDetection.IsFullFramework;
+        protected bool IsUapHandler => !UseManagedHandler && PlatformDetection.IsWindows && PlatformDetection.IsUap;
```

Comment:
Unrelated to this PR, but this makes me wonder: should we allow HttpClientHandler to say what handler it's using under the covers?  Maybe with its ToString returning a description that includes that?

Reply:
Yeah, that does seem like a good idea...

Reply:
Nit: We can use `IsWinRTHandler` for `PlatformDetection.IsUap`